modules/components/src/AdvancedFacetView/FacetView.js

```diff
@@ -105,6 +106,7 @@ export default class FacetView extends React.Component {
                 },
                 () => {
                   onValueChange({ sqon, value });
+                  onTermSelected?.(value);
```

Comment:
why not use the existing `onValueChange`?

Reply:
onTermSelected propagates up from the facet itself from an earlier PR I made #375